test/e2e.method.signing.js

```diff
@@ -0,0 +1,141 @@
+var assert = require('assert');
+var EJSCommon = require('ethereumjs-common');
+var EJSTx = require('ethereumjs-tx');
+var Web3 = require('../packages/web3');
+var Basic = require('./sources/Basic');
+var utils = require('./helpers/test.utils');
+var util = require('util')
+
+describe('transaction and message signing [ @E2E ]', function() {
+    var web3;
+    var accounts;
+    var wallet;
+    var Transaction = EJSTx.Transaction;
+    var Common = EJSCommon.default;
+
+
+    before(async function(){
+        web3 = new Web3('http://localhost:8545');
+        accounts = await web3.eth.getAccounts();
+
+        // Create a funded account w/ a private key
+        wallet = web3.eth.accounts.wallet.create(10);
+
+        await web3.eth.sendTransaction({
+            from: accounts[0],
+            to: wallet[0].address,
+            value: web3.utils.toWei('50', 'ether'),
+        });
+    })
+
+    it('sendSignedTransaction (with ethereumjs-tx@2.x)', async function(){
```

Comment:
No longer required because web3.js is using `ethereumjs-tx` to sign transactions.(PR: #3125)